config/locales/js-en.yml

```diff
@@ -82,6 +83,7 @@ en:
     label_collapse: "Collapse"
     label_collapsed: "collapsed"
     label_collapse_all: "Collapse all"
+    label_committed_at: "committed revision %{revision}"
```

Comment:
You're right, I removed `at` from the value simply because the whole label becomes the link text, and at shouldn't be part of it. 

It could also be that the link text is still subject to change, e.g., so that only the `revision %{revision}` part is clickable. Do you have any input on that?